lib/TPP/Dialect/Tpp/TppUtils.cpp

```diff
@@ -308,15 +308,45 @@ bool canMapToTppAdd(linalg::GenericOp linalgOp) {
   return hasOnlyScalarElementwiseOp<arith::AddFOp>(linalgOp.getRegion());
 }
 
-// TODO: check access pattern using affine map.
```

Comment:
Is this TODO not relevant any more? Doesn't seem to be what you're implementing in this patch. Same for the one below that you also removed.

Reply:
I wanted to check the affine maps to be projected permutations, see: `allIndexingsAreProjectedPermutation`